.github/workflows/stackhpc.yml

```
@@ -178,12 +178,13 @@ jobs:
           ansible-playbook -v ansible/site.yml
           ansible-playbook -v ansible/ci/check_slurm.yml
 
-      - name: Test reimage of compute nodes and compute-init (via rebuild adhoc)
+      - name: Test compute node reimage, compute-init, and reboot
```

```suggestion
      - name: Test compute node reboot and compute-init
```
Currently doesn't test an actual rebuild